staging/src/k8s.io/pod-security-admission/metrics/metrics.go

```diff
@@ -16,9 +16,54 @@ limitations under the License.
 
 package metrics
 
+import (
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	admissionapi "k8s.io/pod-security-admission/admission/api"
+)
+
+const (
+	namespace = "pod_security"
+)
+
+var SecurityEvaluation = metrics.NewCounterVec(
+	&metrics.CounterOpts{
+		Namespace:      namespace,
+		Name:           "evaluations_total",
+		Help:           "Counter of pod security evaluations.",
```

Comment:
Add {"decision", "policy_level", "policy_version", "mode", "operation", "resource"} in help messages.

An example: https://github.com/kubernetes/kubernetes/blob/v1.21.1-rc.0/staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L75-L82